app/controllers/provider_interface/provider_users_controller.rb

```diff
@@ -1,7 +1,7 @@
 module ProviderInterface
   class ProviderUsersController < ProviderInterfaceController
     before_action :require_feature_flag
-    before_action :redirect_unless_permitted_to_manage_users
+    before_action :require_manage_users_permission
```

Comment:
These don't have to change once https://github.com/DFE-Digital/apply-for-teacher-training/pull/2543 is merged.